modules/label/modules/barcode/commerce_pos_label_barcode.module

```diff
@@ -71,19 +74,28 @@ function commerce_pos_label_barcode_generate($product, $format) {
   $generator = new BarcodeGeneratorPNG();
   $upc_field = variable_get('commerce_pos_label_barcode_upc_field_' . $product->type, FALSE);
 
-  if (!empty($upc_field) && $field_upc = field_get_items('commerce_product', $product, $upc_field)) {
-    try {
-      $barcode_settings = $format['barcode'];
-
-      $barcode =  $generator->getBarcode($field_upc[0]['value'], $barcode_settings['type'], $barcode_settings['widthFactor'], $barcode_settings['totalHeight'], $barcode_settings['color']);
-      return $barcode;
+  if(!empty($upc_field)) {
```

Comment:
There should be a space in between your if statement and the opening parentheses.